src/models/InviteCode.ts

```diff
@@ -1,7 +1,27 @@
 import { HexString } from '../helpers/opaqueTypes';
 
+export const enum InviteCodeFields {
```

Comment:
I get errors like this from the Metro bunder on this line: 
`'const' enums are not supported.`

Interestingly the `npm run check` does not report an error on this.

Reply:
yes, it is caused by babel